README.md

```diff
@@ -100,8 +100,8 @@ You can [view the source for the installer script](./deployments/installer/insta
 and use it on your hosts by running:
 
 ```sh
-curl -sSL https://dl.signalfx.com/signalfx-agent.sh > /tmp/signalfx-agent.sh
-sudo sh /tmp/signalfx-agent.sh --realm YOUR_SIGNALFX_REALM -- YOUR_SIGNALFX_ACCESS_TOKEN
```

Comment:
I recently changed this to `YOUR_SIGNALFX_ACCESS_TOKEN` since we require the access token for sending datapoints, which is different than the API token.

Reply:
Just FYI - `YOUR_SIGNALFX_API_TOKEN` is a keyword for the UI to auto-replace with the current user's API token. `YOUR_SIGNALFX_ACCESS_TOKEN` has no auto-replace functionality, so maybe there is a better option than using this text. Although it does get the point across.

Reply:
I'm a little confused. The following is what we currently have in the doc and UI, and it seems to populate content correctly. Does this need to change? 

`--realm YOUR_SIGNALFX_REALM -- YOUR_SIGNALFX_API_TOKEN`

` --realm us0 -- rb2yT1_ej4yXYkIdSQzp_w`

Reply:
> Just FYI - `YOUR_SIGNALFX_API_TOKEN` is a keyword for the UI to auto-replace with the current user's API token. `YOUR_SIGNALFX_ACCESS_TOKEN` has no auto-replace functionality, so maybe there is a better option than using this text. Although it does get the point across.

Thanks @mstumpfx! I was told to change it in my last PR and didn't realize it was a keyword for auto-replacement.

Reply:
> I'm a little confused. The following is what we currently have in the doc and UI, and it seems to populate content correctly. Does this need to change?
> 
> `--realm YOUR_SIGNALFX_REALM -- YOUR_SIGNALFX_API_TOKEN`
> 
> ` --realm us0 -- XXXXXXXXXXX`

@galpizarsfx Keep your changes. It should be `YOUR_SIGNALFX_API_TOKEN`.

Reply:
`YOUR_SIGNALFX_API_TOKEN` should populate correctly with the current user's API token when logged in and viewing inside the application.

However, it sounds like when running the install script we need an _access_ token, not the api token. Which would mean we are incorrectly substituting here. But I think this is the part I'm unsure of - if the token that is displayed is valid or not for the install script. It sounds like @jchengsfx was saying we cannot use an API token here.

`YOUR_SIGNALFX_ACCESS_TOKEN` does not auto-replace with anything and would be displayed as is. But this could be valid way to display this message still, since we need the user to manually substitute with an access token.

Reply:
Okay just kidding, `YOUR_SIGNALFX_API_TOKEN` actually is being replaced by the orgs default access token, _not_ the user API token. My bad. Keep your changes, all good.